connectors/kibana.py

```diff
@@ -136,7 +133,7 @@ async def prepare(service_type, index_name, config, filtering=None):
 
     try:
         # https:#github.com/elastic/enterprise-search-team/discussions/2153#discussioncomment-2999765
-        doc = {
+        doc = connector_definition or {
```

Comment:
do you think we should do something like:
```python
connector_definition["pipeline"] = {
    "extract_binary_content": True,
    "name": "ent-search-generic-ingestion",
    "reduce_whitespace": True,
    "run_ml_inference": True,
}
```

Or we need to `put_pipeline` for the pipeline from `connector_definition`?

Reply:
Sorry, I don't understand it - for now pipeline is both expected in `connector.json` and is present in the default definition, why do we need to override the pipeline?

Reply:
Hmmm maybe, but not now - I don't think we also allow changing the pipeline anywhere in UI, do we?

I thought of doing it later.

Reply:
> Do you have the plan to enable connectors to specify different pipelines?

Not for now at least, we haven't discussed this at all. I think as soon as we need this capability, we can add the logic! For now if someone specifies invalid pipeline it will probably fail the test (need to test it actually)